tests/integration/test_notebooks_pyspark.py

```diff
@@ -11,7 +11,7 @@
 
 
 @pytest.mark.spark
-@pytest.mark.integration
+#@pytest.mark.integration
```

Comment:
this will work for our case, but I think a more explicit way to handle this would be
```
@pytest.mark.skip(reason="disabling test with large datasets")
```